setup.py

```diff
@@ -149,4 +149,5 @@ def get_long_description():
               },
           zip_safe=False,
           test_suite='selftests',
-          python_requires='>=2.6')
+          python_requires='>=2.6',
+          install_requires=['stevedore'])
```

Comment:
Actually after reading https://packaging.python.org/requirements/ I think we should list all the essential packages required for avocado, until we make them optional. Also we should define the version as eg. we do require certain version of stevedore (not sure if conditional versions are supported, but we can add it conditionally).